.github/actions/e2e/action.yaml

```diff
@@ -10,9 +10,9 @@ inputs:
   token_secret:
     required: true
     description: "The clickhouse token secret"
-  token:
```

Comment:
I believe this is required, no?

Reply:
The token? It was an internal e2e thing used to group services by e2e run. It's replaces by `service_name` below